sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Samples/Sample2_AnalyzeConversation_OrchestrationPrediction.cs

```diff
@@ -50,29 +50,28 @@ public void AnalyzeConversationOrchestrationPredictionQuestionAnswering()
 
             Response response = client.AnalyzeConversation(RequestContent.Create(data));
 
-            using JsonDocument result = JsonDocument.Parse(response.ContentStream);
-            JsonElement conversationalTaskResult = result.RootElement;
-            JsonElement orchestrationPrediction = conversationalTaskResult.GetProperty("result").GetProperty("prediction");
+            dynamic conversationalTaskResult = response.Content.ToDynamic();
+            dynamic orchestrationPrediction = conversationalTaskResult.result.prediction;
             #endregion
 
             #region Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPredictionQnA
-            string respondingProjectName = orchestrationPrediction.GetProperty("topIntent").GetString();
-            JsonElement targetIntentResult = orchestrationPrediction.GetProperty("intents").GetProperty(respondingProjectName);
+            string respondingProjectName = orchestrationPrediction.topIntent;
+            dynamic targetIntentResult = orchestrationPrediction.intents[respondingProjectName];
 
-            if (targetIntentResult.GetProperty("targetProjectKind").GetString() == "QuestionAnswering")
+            if (targetIntentResult.targetProjectKind == "QuestionAnswering")
             {
                 Console.WriteLine($"Top intent: {respondingProjectName}");
 
-                JsonElement questionAnsweringResponse = targetIntentResult.GetProperty("result");
+                dynamic questionAnsweringResponse = targetIntentResult.result;
                 Console.WriteLine($"Question Answering Response:");
-                foreach (JsonElement answer in questionAnsweringResponse.GetProperty("answers").EnumerateArray())
+                foreach (dynamic answer in questionAnsweringResponse.answers)
                 {
-                    Console.WriteLine(answer.GetProperty("answer").GetString());
+                    Console.WriteLine(answer.answer?.ToString());
                 }
             }
             #endregion
 
-            Assert.That(targetIntentResult.GetProperty("targetProjectKind").GetString(), Is.EqualTo("QuestionAnswering"));
+            Assert.That(targetIntentResult.targetProjectKind?.ToString(), Is.EqualTo("QuestionAnswering"));
```

Comment:
```suggestion
            Assert.That((string)targetIntentResult.targetProjectKind, Is.EqualTo("QuestionAnswering"));
```

Curious: is there a reason not to use a cast here instead?

Reply:
It can throw. See [comment above](https://github.com/Azure/azure-sdk-for-net/pull/35202#discussion_r1152471558).